test/2-way-jitsi/playbook.json

```diff
@@ -58,33 +51,14 @@
+                    "sdp": "v=0\r\no=mozilla...THIS_IS_SDPARTA-74.0 6213535423809421069 0 IN IP4 0.0.0.0\r\ns=-\r\nt=0 0\r\na=fingerprint:sha-256 BF:9B:12:7E:9D:21:81:C2:24:CB:95:56:64:FF:0A:39:67:76:A9:76:7C:4D:C1:BF:41:A3:F3:78:27:2D:8E:07\r\na=ice-options:trickle\r\na=msid-semantic: WMS *\r\na=group:BUNDLE audio video\r\nm=audio 9 RTP/SAVPF 111 126\r\nc=IN IP4 0.0.0.0\r\na=rtpmap:111 opus/48000/2\r\na=rtpmap:126 telephone-event/8000/1\r\na=fmtp:111 maxplaybackrate=48000;stereo=1;useinbandfec=1\r\na=fmtp:126 0-15\r\na=extmap:1 urn:ietf:params:rtp-hdrext:ssrc-audio-level\r\na=setup:active\r\na=mid:audio\r\na=sendrecv\r\na=ice-ufrag:4b9c330d\r\na=ice-pwd:43b827fc854db60d76148a830db40649\r\na=ssrc:3191347353 cname:{6e2a5bcd-0a9b-de49-8961-6492352cc71a}\r\na=ssrc:3191347353 msid:{73ae53db-3171-4140-a976-106aac04af9e} {6e24624c-64d8-d94b-82bc-f392494d0bcb}\r\na=rtcp-mux\r\nm=video 9 RTP/SAVPF 100\r\nc=IN IP4 0.0.0.0\r\na=rtpmap:100 VP8/90000\r\na=fmtp:100 max-fs=12288;max-fr=60\r\na=rtcp-fb:100 nack\r\na=rtcp-fb:100 nack pli\r\na=rtcp-fb:100 ccm fir\r\na=rtcp-fb:100 goog-remb\r\na=extmap:3 http://www.webrtc.org/experiments/rtp-hdrext/abs-send-time\r\na=setup:active\r\na=mid:video\r\na=sendrecv\r\na=ice-ufrag:4b9c330d\r\na=ice-pwd:43b827fc854db60d76148a830db40649\r\na=ssrc:1943095941 cname:{6e2a5bcd-0a9b-de49-8961-6492352cc71a}\r\na=ssrc:1943095941 msid:{7b604828-f31d-134b-bf30-6569d76f65aa} {bbf03f3e-df08-764d-b693-8644351d8f39}\r\na=ssrc:314782469 cname:{6e2a5bcd-0a9b-de49-8961-6492352cc71a}\r\na=ssrc:314782469 msid:{7b604828-f31d-134b-bf30-6569d76f65aa} {bbf03f3e-df08-764d-b693-8644351d8f39}\r\na=ssrc:1316133583 cname:{6e2a5bcd-0a9b-de49-8961-6492352cc71a}\r\na=ssrc:1316133583 msid:{7b604828-f31d-134b-bf30-6569d76f65aa} {bbf03f3e-df08-764d-b693-8644351d8f39}\r\na=rtcp-mux\r\na=rid:1 send\r\na=rid:2 send\r\na=rid:3 send\r\na=simulcast:send 1;2;3\r\n"
```

Comment:
Here we have a `createAnswer` step without a corresponding `setLocalDescription` step. Is it normal?

Reply:
In TPC, I have removed the unified->plan-b conversions where they were not necessary. We didn't have to do the conversions between createAnswer and setLocalDescription and that is why we don't see it in the call flow.

Reply:
I see, thanks for explaining Jaya!